src/utils/index.ts

```diff
@@ -21,111 +20,34 @@ import {
-      throw new ValidationError(
-        `dropsToXrp: invalid value '${drops}',` +
-          ` should be a number matching (^-?[0-9]*\\.?[0-9]*$).`
-      );
-    } else if (drops === ".") {
-      throw new ValidationError(
-        `dropsToXrp: invalid value '${drops}',` +
-          ` should be a BigNumber or string-encoded number.`
-      );
-    }
-  }
-
-  // Converting to BigNumber and then back to string should remove any
-  // decimal point followed by zeros, e.g. '1.00'.
-  // Important: specify base 10 to avoid exponential notation, e.g. '1e-7'.
-  drops = new BigNumber(drops).toString(10);
-
-  // drops are only whole units
-  if (drops.includes(".")) {
-    throw new ValidationError(
-      `dropsToXrp: value '${drops}' has` + ` too many decimal places.`
-    );
-  }
-
-  // This should never happen; the value has already been
-  // validated above. This just ensures BigNumber did not do
-  // something unexpected.
-  if (!drops.match(/^-?[0-9]+$/)) {
-    throw new ValidationError(
-      `dropsToXrp: failed sanity check -` +
-        ` value '${drops}',` +
-        ` does not match (^-?[0-9]+$).`
-    );
-  }
-
-  return new BigNumber(drops).dividedBy(1000000.0).toString(10);
-}
-
-function xrpToDrops(xrp: BigNumber.Value): string {
-  if (typeof xrp === "string") {
-    if (!xrp.match(/^-?[0-9]*\.?[0-9]*$/)) {
-      throw new ValidationError(
-        `xrpToDrops: invalid value '${xrp}',` +
-          ` should be a number matching (^-?[0-9]*\\.?[0-9]*$).`
-      );
-    } else if (xrp === ".") {
-      throw new ValidationError(
-        `xrpToDrops: invalid value '${xrp}',` +
-          ` should be a BigNumber or string-encoded number.`
-      );
-    }
-  }
-
-  // Important: specify base 10 to avoid exponential notation, e.g. '1e-7'.
-  xrp = new BigNumber(xrp).toString(10);
-
-  // This should never happen; the value has already been
-  // validated above. This just ensures BigNumber did not do
-  // something unexpected.
-  if (!xrp.match(/^-?[0-9.]+$/)) {
-    throw new ValidationError(
-      `xrpToDrops: failed sanity check -` +
-        ` value '${xrp}',` +
-        ` does not match (^-?[0-9.]+$).`
-    );
-  }
-
-  const components = xrp.split(".");
-  if (components.length > 2) {
-    throw new ValidationError(
-      `xrpToDrops: failed sanity check -` +
-        ` value '${xrp}' has` +
-        ` too many decimal points.`
-    );
-  }
-
-  const fraction = components[1] || "0";
-  if (fraction.length > 6) {
-    throw new ValidationError(
-      `xrpToDrops: value '${xrp}' has` + ` too many decimal places.`
-    );
-  }
-
-  return new BigNumber(xrp)
-    .times(1000000.0)
-    .integerValue(BigNumber.ROUND_FLOOR)
-    .toString(10);
-}
-
+/**
+ * TODO: Remove/rename this function.
+ *
+ * @param amount - Convert an Amount in.
+ * @returns Amount without X-Address issuer.
+ * @throws When issuer X-Address includes a tag.
+ */
 function toRippledAmount(amount: RippledAmount): RippledAmount {
```

Comment:
This method shouldn't be needed anymore (idk if it's still used anywhere).

Reply:
This is still used pretty heavily in the old code, so I'm not gonna rip it out til the `src/transaction` directory goes.